client/src/pages/ProfileListings/DateSelectPopover.tsx

```diff
@@ -0,0 +1,135 @@
+import { useState } from 'react';
+import Grid from '@material-ui/core/Grid';
+import Snackbar from '@material-ui/core/Snackbar';
+import Card from '@material-ui/core/Card';
+import CardHeader from '@material-ui/core/CardHeader';
+import CardActions from '@material-ui/core/CardActions';
+import Popover from '@material-ui/core/Popover';
+import CloseIcon from '@material-ui/icons/Close';
+import Typography from '@material-ui/core/Typography';
+import IconButton from '@material-ui/core/IconButton';
+import Button from '@material-ui/core/Button';
+import { MuiPickersUtilsProvider, KeyboardDatePicker } from '@material-ui/pickers';
+import DateFnsUtils from '@date-io/date-fns';
+import useStyles from './useStyles';
+import Alert from './alert';
+
+interface DateSelectProps {
+  open: true | false;
+  handleOpen: any;
```

Comment:
```suggestion
  handleOpen: () => void;
```
Instead of `any`, we can use functional type like this.

Reply:
Awesome -- thanks! I couldn't figure out how to do this.